app/components/SuggestionContents/index.jsx

```diff
@@ -0,0 +1,81 @@
+import TextDiff from '../TextDiff'
+import React from 'react'
+import { IntlMixin } from 'react-intl'
+import cx from 'classnames'
+
+/**
+ * Display all content strings (singular or plurals) for a suggestion.
+ * May show a diff against a set of provided strings.
+ */
+let SuggestionContents = React.createClass({
+  mixins: [IntlMixin],
+
+  propTypes: {
+    plural: React.PropTypes.bool.isRequired,
+    contents: React.PropTypes.arrayOf(
```

Comment:
Is is really required? It will just be false if they don't provide it. Does it need to be explicit every time?